spock-core/src/main/java/org/spockframework/runtime/extension/builtin/RollupExtension.java

```diff
@@ -0,0 +1,69 @@
+/*
+ * Copyright 2020 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.spockframework.runtime.extension.builtin;
+
+import org.spockframework.runtime.InvalidSpecException;
+import org.spockframework.runtime.extension.AbstractAnnotationDrivenExtension;
+import org.spockframework.runtime.extension.AbstractGlobalExtension;
+import org.spockframework.runtime.model.FeatureInfo;
+import org.spockframework.runtime.model.SpecInfo;
+import spock.lang.Rollup;
+import spock.lang.Unroll;
+
+public class RollupExtension extends AbstractAnnotationDrivenExtension<Rollup> {
```

Comment:
Also `@Beta` and `@since 2.0` (for code readers)? Possibly with the link to `@Rollup` to learn more about expected behavior.

Reply:
Added the first two.
Regarding `Rollup`, it is the type parameter for the super class, so it is linked already from IDE and also JavaDoc, so I don't think we need an additional link.